examples/run_ransac.py

```diff
@@ -0,0 +1,85 @@
+"""
+=================
+Run RANSAC
+=================
+
+
+In this example we show how to run the RANSAC of ``pyprep``.
+"""
+
+# Authors: Yorguin Mantilla <yjmantilla@gmail.com>
+#
+# License: MIT
+# Based On: use_noisy_module.py
+
+###############################################################################
+# First we import what we need for this example.
+import numpy as np
+import mne
+from scipy import signal as sx
+import time as clk
```

Comment:
```suggestion
import time
```

please avoid using custom "import names". We can and should use the common ones such as numpy as np, pandas as pd, etc.

and I am fine with "scipy.signal as signal" --> but as soon as we get too funky with these, it will negatively impact readability. E.g., every pythonista knows `time.time()` ... but when they see `clk.time()` they cannot be sure whether this is some custom func, or actually `time.time()` (they first have to look at imports again)